index-transformer.js

```diff
@@ -0,0 +1,13 @@
+import fs from 'fs-extra'
+import Transformer from './test-asset-generator/transformers/transformer'
+
+const assets = []
+
+assets.push(JSON.parse(fs.readFileSync('test-asset-generator/assets/organisations/pterodactyl-laboratories/organisation.json')))
+assets.push(JSON.parse(fs.readFileSync('test-asset-generator/assets/organisations/pterodactyl-laboratories/marketing-authorisations/marketing-authorisation.1.json')))
+assets.push(JSON.parse(fs.readFileSync('test-asset-generator/assets/organisations/pterodactyl-laboratories/marketing-authorisations/marketing-authorisation.2.json')))
+assets.push(JSON.parse(fs.readFileSync('test-asset-generator/assets/organisations/pterodactyl-laboratories/marketing-authorisations/marketing-authorisation.3.json')))
+
+const transformer = new Transformer()
+transformer._constructor(assets)
+console.log(transformer.transform())
```

Comment:
Was this comment intended to be left?

Reply:
This file will be deleted anyway once all the transformer work is complete. We are just using another branch as a base until we're ready to merge into develop.